src/_includes/toc-and-content.html

```diff
@@ -0,0 +1,58 @@
+<div class="row">
+  <div class="toc col-md-3">
+    <div class="toc-container">
+      <div class="d-none d-sm-none d-md-block d-lg-block">
+        <h2>Table of Contents</h2>
+        {% for item in include.toc_data %}
+        <h3>{{ item.title }}</h3>
+        <ul>
+          {% for subsection_item in item.subsection_items %}
+          <li class="h5">
+            <a href="#{{ subsection_item.url }}">{{ subsection_item.title }}</a>
+          </li>
+          {% endfor %}
+        </ul>
+        {% endfor %}
+      </div>
+      <div id="mobile-toc-nav" class="d-block d-sm-block d-md-none d-lg-none">
+        <div class="mobile-control-group main-nav-icons toc-and-content-mobile-nav">
+          <button
+            id="mobile-toc-nav-button"
+            class="mobile-control toggle-menu"
+            aria-expanded="false"
+            aria-controls="navigation"
+            data-toggle="collapse"
+            data-target="#navigation-two"
+          >
+            <h2>Table of Contents</h2>
+          </button>
+        </div>
+        <nav id="navigation-two" class="main-navigation singlelevel auto-highlight no-icons nav collapse">
+          <div id="nav_list" class="top-level-nav">
+            {% for item in include.toc_data %}
+            <h3>{{ item.title }}</h3>
+            <ul>
+              {% for subsection_item in item.subsection_items %}
+              <li class="h5">
+                <a class="mobile-nav-link" href="#{{ subsection_item.url }}">{{ subsection_item.title }}</a>
+              </li>
+              {% endfor %}
+            </ul>
+            {% endfor %}
+          </div>
+        </nav>
+      </div>
+    </div>
+  </div>
+  <div class="col-md-9 content-container">
+    {% for item in include.toc_data %}
+    <h3>{{ item.title }}</h3>
```

Comment:
```suggestion
    <h2>{{ item.title }}</h2>
```

Reply:
Since each of these top-level TOC items represents a new section, and otherwise we skip from `<h1>` --> `<h3>`